src/mono/mono/mini/mini-arm64.c

```diff
@@ -4083,17 +4090,19 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 
 		case OP_ARM64_XADDV: {
 			switch (ins->inst_c0) {
-			case INTRINS_AARCH64_ADV_SIMD_FADDV:
+			case INTRINS_AARCH64_ADV_SIMD_FADDV: {
+				const int width = get_vector_size_macro (ins);
 				if (ins->inst_c1 == MONO_TYPE_R8) {
 					arm_neon_faddp (code, VREG_FULL, TYPE_F64, dreg, sreg1, sreg1);
```

Comment:
If working on 64b vectors of `double`, this should just be a `mov dreg, sreg1` if `dreg != sreg1` or nothing if they are the same. Doing this `addp` will add the only `double` in the "vector" with an undefined value in the upper part of the vector.

Reply:
You are right. I did not test this PR for Vector64 but only for Vector2 which has only `float` elements. I will put a comment to the code to fix this when enabling Vector64.